instruments/thorlabs/pm100usb.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, parent):
             self._parent = parent
 
             # Pull details about the sensor from SYST:SENSOR:IDN?
-            sensor_idn = parent.sendcmd("SYST:SENSOR:IDN?")
+            sensor_idn = parent.query("SYST:SENSOR:IDN?")
```

Comment:
Instrument inherits from `SCPIInstrument`, which inherits from `Instrument`. The Thorlabs PMC100USB is, according to the docstring, SCPI-compliant, so this request should be a `query` and not a `sendcmd`.

Reply:
Not the exact manual, but found [the one for the PM100D](https://www8.physics.utoronto.ca/~phy326/opt/PM100D-Manual.pdf): Confirms that this is a query only command. Switching it from `sendcmd` to `query` should be correct.